src/Microsoft.Framework.Cache.Memory/IMemoryCache.cs

```diff
@@ -2,11 +2,15 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+#if !NET45
 using Microsoft.Framework.Runtime;
+#endif
 
 namespace Microsoft.Framework.Cache.Memory
 {
+#if !NET45
```

Comment:
Can we change these to `#if ASPNET50 || ASPNETCORE50` instead. If in future we extend this more frameworks we don't have to change this.